### DISCUSSION TITLE
How to enable Dynamic rules?

### DISCUSSION THREAD

**Discussion by dhiren-shah** [shlinkio/shlink#2094] [Help wanted] [answered]
### Shlink version

4.03.

### PHP version

8.3

### How do you serve Shlink

Self-hosted RoadRunner

### Database engine

PostgreSQL

### Database version

16

### Summary

Hi,
Thanks for developing this application. It is working smoothly on my local.
But I could not find the option to enable the Dynamic rule-based redirects as mentioned on this document https://shlink.io/documentation/advanced/dynamic-rule-based-redirects-system/

**Comment by acelaya** [comment, marked as answer]
There's nothing to "enable". When Shlink matches a short URL and is going to determine where to redirect, it first checks if the short URL has redirect rules attached to it, and evaluates them in order of priority.

If none of them match, then it redirects to the default long URL.

If by "enabling" you actually mean how do you configure them when using shlink-web-client, there's an option in the dropdown for every short URL, which will take you to a new section to manage the redirect rules.

https://github.com/shlinkio/shlink/assets/2719332/02fa8212-5132-4145-8e9f-59a46a1495fd

**Reply by dhiren-shah** [reply]
Thank you very much, I missed that completely.
